src/components/layouts/MainLayout.jsx

```diff
@@ -1,8 +1,231 @@
+import { Suspense, useState } from 'react'
 import { Outlet } from 'react-router-dom'
+import styled, { css } from 'styled-components'
 
-export const MainLayout = () => (
-  <>
-    main data
-    <Outlet />
-  </>
-)
+import { useTranslation } from 'react-i18next'
+import { BurgerMenuIcon, MoonIcon, SunIcon } from '../../assets/icons'
+import { useAppThemeContext } from '../../styles/theme/provider'
+import { SideBar } from '../SideBar'
+import {
+  useBreakpointValue,
+  ScreenWrapper,
+  getBreakpointsStyles,
+  getDesktopStyles,
+} from '../../styles/breakpoints'
+import { PrimaryColorBtn } from '../PrimaryColorBtn'
+import { Text } from '../Text'
+
+export const MainLayout = () => {
+  const { t } = useTranslation()
+  const [isBurgerMenuOpen, setIsBurgerMenuOpen] = useState(false)
+  const [selectedTab, setSelectedTab] = useState('calendar')
+  const { themeType, setThemeType } = useAppThemeContext()
+  const iconSize = useBreakpointValue({ mobileValue: 24, tabletValue: 32, desktopValue: 32 })
+  const nameFontSize = useBreakpointValue({
+    mobileValue: 14,
+    tabletValue: 18,
+    desktopValue: 18,
+  })
+
+  const userName = 'Yuliia'
+
+  const handleThemeChange = () => {
+    setThemeType(themeType === 'light' ? 'dark' : 'light')
+  }
+
+  const firstLetersMaker = (str) => {
+    const words = str.split(' ')
+    const capitalizedWords = words
+      .map((word) => {
+        return word.split('')[0].toUpperCase()
```

Comment:
word[0].toUpperCase() -> no need to split and join
No need return here

CapitalizedLetters - actually it's no workds now)

And before returning you need to slice it.
Because if you will have this name "A W F D X" -> you will receive "AWFDX" but we need max 2 letters